turbinia/state_manager.py

```diff
@@ -299,6 +299,12 @@ def update_task(self, task):
     if not key:
       self.write_new_task(task)
       return
+    log.warn(f'updating task {task.__dict__}')
```

Comment:
Please remove all log statements used for debugging / troubleshooting purposes during development.